### DISCUSSION TITLE
Welcome to Bloom Filter Implementation Discussions!

### DISCUSSION THREAD

**Discussion by Clare-Joyce** [Clare-Joyce/CDS_Project#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to discuss our ideas starting with discussions around understanding what a **Bloom Filter** is, then moving on to understanding its algorithm. This will then be followed by the implementation and at this stage, this will serve as a platform to discuss ideas on how to implement and enhance our implementation. This page is meant for you to:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with your team member.

Let's make this happen!

**Comment by Sarigakakkamani** [comment]
Thank you for initiating the discussion. 
Let's plan to regroup next Monday, April 1st, 2024, to discuss our progress and exchange ideas.
